docs/getting-started-guides/dcos.md

```diff
@@ -0,0 +1,175 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/getting-started-guides/dcos.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+Getting started with Kubernetes on DCOS
+----------------------------------------
+
+**Table of Contents**
+
+- [About Kubernetes on DCOS](#about-kubernetes-on-dcos)
+- [Prerequisites](#prerequisites)
+- [Install Kubernetes](#install-kubernetes)
+- [Operations](#operations)
+- [Uninstall Kubernetes](#uninstall-kubernetes)
+
+
+## About Kubernetes on DCOS
+
+The [Datacenter Operating System (DCOS)](https://docs.mesosphere.com/) is a collection of tools for managing distributed systems deployed on a cluster of machines (virtual or bare metal).
+
+Since Kubernetes can be configured to schedule through Mesos, the cluster kernel at the heart of DCOS, it's a natural fit to deploy Kubernetes on DCOS.
+
+The recommended way to install Kubernetes on DCOS is to use the latest pre-packaged releases in the [Mesosphere Multiverse](https://github.com/mesosphere/multiverse) DCOS package repository. This package contains a [Marathon](https://mesosphere.github.io/marathon/) deployment manifest that specifies how to run the Kubernetes core components in containers deployed and monitored by Marathon, the DCOS init system.
```

Comment:
I was thinking of just dropping the "recommend" and going with "To install Kubernetes on DCOS use the latest pre-packaged releases"